Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -241,6 +241,8 @@ internal sealed partial class NodejsPackage : CommonPackage {
             MakeDebuggerContextAvailable();
 
             IntellisenseOptionsPage.AnalysisLogMaximumChanged += IntellisenseOptionsPage_AnalysisLogMaximumChanged;
+            IntellisenseOptionsPage.AnalysisLevelChanged += IntellisenseOptionsPageAnalysisLevelChanged;
+            IntellisenseOptionsPage.SaveToDiskChanged += IntellisenseOptionsPageSaveToDiskChanged;
```

Comment:
Verified. The callback is only fired when the user goes in to the options page and changes the setting, not on first load.